pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.ruff]
+line-length = 120
+
+[tool.ruff.lint]
+# E, F: Standard pycodestyle and Pyflakes (default flake8)
+# D: pydocstyle (for D100, D101, etc.)
+# Q: flake8-quotes (for Q000)
+# PL: Pylint
+extend-select =[
+    "E", "F", "D", "Q", "PL",
+    "T100", # https://docs.astral.sh/ruff/rules/#flake8-debugger-t10
+    "T2", # https://docs.astral.sh/ruff/rules/#flake8-print-t20
+    "TD", # https://docs.astral.sh/ruff/rules/#flake8-todos-td
+]
+
+ignore = [
+    "E501",   # line-too-long
```

Comment:
Will the check complain if the line is > 120 with this as ignore?

Reply:
closing with a note. ruff will not complain. the style will need to get formatted by tidy.